adobe_vipm/flows/validation/termination.py

```diff
@@ -39,6 +43,8 @@ def validate_termination_order(client, order):
     pipeline = Pipeline(
         SetupContext(),
         ValidateDuplicateLines(),
+        SetOrUpdateCotermNextSyncDates(),
+        ValidateRenewalWindow(is_validation=True),
```

Comment:
Could you please write down and fix it somewhere on next PRs, I would prefer to make this step with an hour parameter for time limit